testsuite/smoke-test/src/txn_emitter.rs

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::smoke_test_environment::new_local_swarm_with_aptos;
+use anyhow::ensure;
+use aptos_sdk::{transaction_builder::TransactionFactory, types::PeerId};
+use forge::{EmitJobRequest, NodeExt, Result, Swarm, TxnEmitter, TxnStats};
+use rand::{rngs::OsRng, SeedableRng};
+use std::time::Duration;
+use tokio::runtime::Builder;
+
+pub async fn generate_traffic(
+    swarm: &mut dyn Swarm,
+    nodes: &[PeerId],
+    duration: Duration,
+    gas_price: u64,
+) -> Result<TxnStats> {
+    ensure!(gas_price > 0, "gas_price is required to be non zero");
+    let mut runtime_builder = Builder::new_multi_thread();
+    runtime_builder.enable_all();
+    runtime_builder.worker_threads(64);
+    let rng = SeedableRng::from_rng(OsRng)?;
+    let validator_clients = swarm
+        .validators()
+        .filter(|v| nodes.contains(&v.peer_id()))
+        .map(|n| n.rest_client())
+        .collect::<Vec<_>>();
+    let fullnode_clients = swarm
```

Comment:
This is not needed as you are not starting any FN.

Reply:
leaving it generic if someone wants to use for local test

Reply:
It is confusing - if we want to keep the FN clients - let's add some full nodes to the swarm?